privacyidea/lib/auditmodules/sqlaudit.py

```diff
@@ -188,9 +216,9 @@ def _create_filter(param, timelimit=None):
                     else:
                         # All other keys are compared as strings
                         column = getattr(LogEntry, search_key)
-                        if search_key == "date":
+                        if search_key.endswith("date"):
```

Comment:
I hope we never have a column or searchkey like "mandate" ;-)

Reply:
Done in fcff8608